src/script.js

```diff
@@ -200,6 +209,176 @@
 			a.remove();
 		});
 
+		        var collapser = (function() {
+            
+        //function to check if only one button of the all is is 'ticked'
+            
+            function onlyOneIsTicked() {
+                var count = 0;
+                collapseBtns.forEach(function(collapseBtn) {
+                    if (collapseBtn.classList.contains("ticked")) {
+                        count += 1;
+                    }
+                });
+                
+                return count === 1;
+            }
+            
+            
+            function manageCollapse(isTicked, pane) {
```

Comment:
this function is too big considering the required functionality. I'll go over the logic but I'll recommend refactoring this function. I am sure it has lots of duplicate code, redundant logic and can be made shorter :)